assets/enums/sample_host.json

```
@@ -18,6 +18,7 @@
         "Rangifer tarandus",
         "Canis lupus",
         "Mammuthus primigenius",
+        "Mammuthus trogontherii",
```

```suggestion
        "Mammuthus sp.",
```

There is no Mammuthus trogontherii unfortunately, so we go with unspecified